src/inc/request-handler.ts

```diff
@@ -5,7 +5,7 @@ export default class RequestHandler {
   constructor (public $auth: Auth) {}
 
   _needToken (config: HTTPRequest) {
-    const options = this.$auth.strategy.options;
+    const options = this.$auth.strategy.options
     return options.token.global || Object.values(options.endpoints).includes(config.url)
```

Comment:
Sometimes the `endpoint` can be an object that have `url` property, like in `local` scheme. So we need to check both :)
```suggestion
    return options.token.global || Object.values(endpoints).some(endpoint => {
      if(typeof endpoint === 'object') {
        return endpoint.url === config.url
      }

      return endpoint === config.url
    })
```

Reply:
Sure ! 
But I still think we can make it cleaner 🤔